streams/src/main/java/org/apache/kafka/streams/kstream/internals/TimeWindowedKStreamImpl.java

```diff
@@ -53,13 +53,13 @@
                             final String name,
                             final Serde<K> keySerde,
                             final Serde<V> valSerde,
-                            final boolean repartitionRequired,
+                            final GroupedStreamAggregateBuilder<K, V> aggregateBuilder,
```

Comment:
I did this to be consistent  with `SessionWindowedKStreamImpl`

Reply:
Thank you!